Documentation missing endpoint explanation (#918)
graphql-python/graphene-django: docs/testing.rst

If you want to unittest your API calls derive your test case from the class `GraphQLTestCase`.

> Your endpoint is set with `GRAPHQL_URL`. The default endpoint is `GRAPHQL_URL="/graphql/"` in `GraphQLTestCase`

Reviewer: ```suggestion
Your endpoint is set through the `GRAPHQL_URL` attribute on `GraphQLTestCase`. The default endpoint is `GRAPHQL_URL = "/graphql/"`.
```

Author: Sounds correct now ;)
